third_party/sl4a/README

```diff
@@ -1,4 +1,4 @@
 This folder a heavily modified version of sl4a which acts as an RPC library for
-third-party facades.
+third-party code snippets (facades).
```

Comment:
Deleted README and merged its content into README.google, where the differences between sl4a and the snippet lib have to be documented anyway. ptal.